jsScripts/formValidation.js

```diff
@@ -0,0 +1,14 @@
+const errorMsg = document.getElementById('validation');
+const formEmailInput = document.querySelector('form input[type="email"]');
+const formBtn = document.querySelector('form button[type="submit"]');
+
+function emailValidation(e) {
+  e.preventDefault();
+  if (formEmailInput.value !== formEmailInput.value.toLowerCase()) {
+    errorMsg.textContent = 'Form not submitted. Email should be lower-cased.';
+  } else {
+    errorMsg.textContent = '';
+  }
```

Comment:
- [ ] You did a great job of validating the email and preventing the form submission. But the form is not getting submitted even when I have a lowercase email value. Please work on this issue. 🤝️